app/Http/Controllers/Table/EventlogController.php

```diff
@@ -119,15 +120,15 @@ private function formatDatetime($eventlog)
     private function severityLabel($eventlog_severity)
```

Comment:
As @Jellyfrog commented here https://github.com/librenms/librenms/pull/11607#discussion_r425115063 maybe that function should go public and common so syslog can use it too?

Reply:
The Class is public and can be used from everywhere.
Class is "type" of enum and each "value" is static so it can be used from everywhere without instancing, as it should be.
(Can be seen on to other changes in PR. Especially on stuff which isn't still Laravel Code)

I know there is much more which could/should be enumerated. That'S the cause of:
LibreNMS/*Enum*/Alert.php so more enum could be added and it stays clear for what this Enum stands for.

Reply:
aaahhh now in know what you meant! you're talking about stuff outside of my PR ...
Sure such function should be public, but i think it should defined also global then ... but that's not the stuff my PR is about ;-)